<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FORM_HydraulicQuery</class>
 <widget class="QWidget" name="FORM_HydraulicQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-image: url(:/pic_de260/bg_01.png);*/

QLabel{
color: rgb(15, 175, 252);
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
}


QLabel#value1{
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
color: rgb(115, 210, 22);
}

QLabel#value2{
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
color: rgb(115, 210, 22);
}

QLabel#value3{
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
color: rgb(115, 210, 22);
}

QLabel#value4{
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
color: rgb(115, 210, 22);
}

QLabel#value5{
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
color: rgb(115, 210, 22);
}

QLabel#value6{
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
color: rgb(115, 210, 22);
}

QLabel#value7{
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
color: rgb(115, 210, 22);
}

QLabel#value8{
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
color: rgb(115, 210, 22);
}

QLabel#value9{
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
color: rgb(115, 210, 22);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>50</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TakaoPGothic</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(252, 233, 79);
font: 30pt &quot;TakaoPGothic&quot;;</string>
   </property>
   <property name="text">
    <string>液压系统</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_6">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>730</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
color: rgb(15, 175, 252);
font: 15pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>里程油耗</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/ico_260/903.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>324</y>
     <width>422</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="value9_2">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>831</x>
     <y>239</y>
     <width>250</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>举升阀压力:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value3">
   <property name="geometry">
    <rect>
     <x>1101</x>
     <y>239</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>0 bar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>831</x>
     <y>196</y>
     <width>250</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>转向蓄能器压力:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value2">
   <property name="geometry">
    <rect>
     <x>1101</x>
     <y>196</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>0 bar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>831</x>
     <y>153</y>
     <width>250</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>转向阀压力:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value1">
   <property name="geometry">
    <rect>
     <x>1101</x>
     <y>153</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>0 bar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1271</x>
     <y>240</y>
     <width>250</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>前桥制动蓄能器压力:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value4">
   <property name="geometry">
    <rect>
     <x>1541</x>
     <y>240</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>0 bar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1271</x>
     <y>283</y>
     <width>250</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>后桥制动蓄能器压力:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value5">
   <property name="geometry">
    <rect>
     <x>1541</x>
     <y>283</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>0 bar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1271</x>
     <y>153</y>
     <width>250</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>前桥制动压力:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value6">
   <property name="geometry">
    <rect>
     <x>1541</x>
     <y>153</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>0 bar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1271</x>
     <y>196</y>
     <width>250</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>后桥制动压力:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value7">
   <property name="geometry">
    <rect>
     <x>1541</x>
     <y>196</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>0 bar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>831</x>
     <y>325</y>
     <width>250</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>液压油箱油温:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value8">
   <property name="geometry">
    <rect>
     <x>1101</x>
     <y>325</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0 ℃</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>831</x>
     <y>283</y>
     <width>250</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>驻车制动压力:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value9">
   <property name="geometry">
    <rect>
     <x>1101</x>
     <y>283</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string>0 bar</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
